xen/arch/arm/ffa.c

```diff
@@ -283,7 +399,11 @@ uint32_t ffa_get_call_count(void)
     uint32_t count = 0;
 
     if ( IS_ENABLED(CONFIG_FFA) )
-        count += 2;
+    {
```

Comment:
I think here we should put the final value we will support as define instead of increasing this at each patch

Reply:
Yeah, it's painful when reworking the patches, but it is disturbing with a mismatch too when reviewing.